pkg/network/protocols/telemetry/metric.go

```diff
@@ -9,34 +9,37 @@ import (
 	"encoding/json"
 	"strings"
 
-	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"go.uber.org/atomic"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
we have StringsSet in the project, unless there is a functionality required that is missing, I think we should stick with less type duplications

Reply:
I'm not adding a new dependency to the project and our own implementation of StringSet doesn't have everything I need such as `IsSuperSet`